alfresco-transform-misc/alfresco-transform-misc-boot/src/test/java/org/alfresco/transformer/MiscTransformerHttpRequestTest.java

```diff
@@ -1,48 +1,48 @@
-/*

```

Comment:
Format only changes should be done in separate PRs - did anything else change other than line-endings (cc @eknizat @PunchedChimera ) ?

Reply:
There are no changes.

Reply:
If no changes then any idea why it is appearing as a diff in this PR ? 

Eric also mentioned that when "misc" was originally added it used CRLF unlike rest of project that uses just LF. 

If so, we could do a sweep (preferably as a separate PR) to make this consistent and reduce likelihood of format-only changes due to git / ide settings ?

Reply:
I've noticed that my git has the following config: `core.autocrlf=true`
If there were areas using LF then this is why we are seeing the changes.

Reply:
OK, thanks. Will you fix now (as part of this PR) or later to be consistent with rest of project ?

Reply:
line changes fixed now set to CRLF